api/proto/wso2/health/service/health.proto

```diff
@@ -24,7 +23,6 @@ message HealthCheckResponse {
 }
 
 // [#protodoc-title: Health]
-service Health { //TODO: (renuka) rename to HealthService ?
```

Comment:
Keep the service name

Reply:
Fixed with https://github.com/wso2/product-microgateway/pull/2805/commits/10af9235789ebfcd2591539f0815d15fe7f622a4. Thanks